localenv/README.md

```diff
@@ -139,6 +139,28 @@ Note that you have to go through an additional "login" step by providing you IPv
 
 After stopping the script it is necessary to clear the environment using the command described in [Shutting down](#Shutting-down). This is necessary as on a new run of the scripts (with autopeering or not) the wallet address url will differ.
 
+### Debugging
+
+Debuggers for the services are exposed on the following ports:
+
+| IP and Port    | Services                |
+| -------------- | ----------------------- |
+| 127.0.0.1:9229 | Cloud Nine Backend      |
+| 127.0.0.1:9230 | Cloud Nine Auth         |
+| 127.0.0.1:9231 | Happy Life Bank Backend |
+| 127.0.0.1:9232 | Happy Life Bank Auth    |
+
+To use these debuggers with a chromium browser:
+
+- go to `chrome://inspect` in chromium browser
+- Click "Configure" add the IP addresses and ports detailed above
+- start docker containers
+- click "inspect" on the service you want to debug to open the chromium debugger
+
+You can either trigger the debugger by adding `debugger` statements in code and restarting, or by adding breakpoints directly in the chromium debugger after starting the docker containers.
+
+For more ways to connect to these debuggers, see the Node docs for debugging: https://nodejs.org/en/learn/getting-started/debugging
+
```

Comment:
I wanted to provide specific instructions for a basic way to use the debugger despite the fact that everyone is liable to have a different setup (vscode, jetbrains, text editor, etc.). Thus I provided instructions for setting up the chromium debugging client and left a link to the node docs if people want to tune it more for their own setup. 

Of course this does require a chromium browser but it's the best we can do. Note that it's not possible to use safari, firefox, or any non-chromium browser because node uses the v8 engine, and those browsers do not.

Reply:
Please let me know if you can follow these instructions to start using the debugger successfully.